### DISCUSSION TITLE
Best Practise for Links?

### DISCUSSION THREAD

**Discussion by arresse** [freeplane/freeplane#1029] [Help] [answered]
Hi there,

Node ID and URI?  What's the difference and how can each be used for creating links within and without FP?

Thanks.

Freeplane is an amazing app!  :-)

**Comment by theelectricsoul** [comment, marked as answer]
Check documentation (F1) :)
![image](https://user-images.githubusercontent.com/10260832/221383015-6550af3f-0962-49d8-b7aa-67bd15ff4576.png)

**Reply by arresse** [reply]
Thank you!
